src/App.js

```diff
@@ -0,0 +1,44 @@
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { fetchTutors } from './redux/tutors/tutors';
+import './App.css';
+import Nav from './modules/Nav';
+import Home from './modules/Home';
+import Login from './modules/user-sessions/login';
+import Signup from './modules/user-sessions/signup';
+import AddTutor from './modules/user-actions/AddTutor';
+import DeleteTutor from './modules/user-actions/DeleteTutor';
+import SingleTutor from './modules/SingleTutor';
+import Reserve from './modules/Reserve';
+import { userSession } from './redux/user/session-redux';
+import Reservations from './modules/Reservations';
+
+function App() {
```

Comment:
Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```



## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/223426154-12c37c30-d030-4e61-bd19-c83024199ca2.png)